Source/sourcekitten/Doc.swift

```diff
@@ -0,0 +1,71 @@
+import ArgumentParser
+import Foundation
+import SourceKittenFramework
+
+extension SourceKitten {
+    struct Doc: ParsableCommand {
+        static let configuration = CommandConfiguration(abstract: "Print Swift or Objective-C docs as JSON")
+
+        @Option(help: "Only document one file")
+        var singleFile: Bool = false
+        @Option(help: "Name of Swift module to document (can't be used with `--single-file`)")
+        var moduleName: String = ""
+        @Option(help: "Document a Swift Package Manager module")
+        var spm: Bool = false
+        @Option(help: "Document Objective-C headers instead of Swift code")
+        var objc: Bool = false
+        @Option(help: "Equivalent to --spm --module-name (string)")
+        var spmModule: String = ""
```

Comment:
Could take the opportunity of a breaking release to delete the old `--spm-module` -- not a big deal either way!

Reply:
Good idea! Done in 032f2e5.